### ISSUE TITLE
Critical regression bug regarding Activate

### ISSUE THREAD

**Issue by AndersMalmgren** [ValveSoftware/steamvr_unity_plugin#582]
Hello. We have this little code block that used to work just fine

```csharp
        public bool UpdateCanMove(NVRHand lastMovementHand)
        {
            var isLastMovementHand = lastMovementHand == this;
            var prio = isLastMovementHand ? 2 : 0;

            if (prio != lastPrio)
            {
                SteamVR_Actions.Movement.Activate(SteamInputSource, prio);
                lastPrio = prio;
            }

            return isLastMovementHand || SteamVR_Actions.Movement.DirectionTouched.GetActive(SteamInputSource);
        }
```

This code checks if you are already moving using this controller (touchpad locomotion). And if you it upgrades the movement action set to prio 2. This way you can grab your firearm which as prio 0 on its action set and still move with the toughpad even if you have actions bound to touchpad. 

Used to work perfect. But now each other frame `SteamVR_Actions.Movement.DirectionTouched.GetActive(SteamInputSource);` return false and the player stops, in next frame he starts to walk again. Resulting in strobe effect. 

I have tried to remove `SteamVR_Actions.Movement.Activate(SteamInputSource, prio);` which fixes the problem. But now I can not pick up items without stop moving

**Comment by AndersMalmgren** [comment]
its related to this issue, https://github.com/ValveSoftware/openvr/issues/1020

**Comment by AndersMalmgren** [comment]
I have been in discussion with @keithbradner over at discord on this issue, but I havent been able to reach him for two days now. Really need someone to take alook at this since this regression totally breaks our game for Vive Wand users

Here is a repro with two different scenarios that break. You can find them in repro.cs

I removed SteamVR for size
[SteamVR_Repro.zip](https://github.com/ValveSoftware/steamvr_unity_plugin/files/3838409/SteamVR_Repro.zip)

**Comment by AndersMalmgren** [comment]
Bug confirmed by other players. Please Valve, this need super high attention. This regression completely breaks game for Vive Wand players

**Comment by AndersMalmgren** [comment]
I have been downloading old versions of steamvr using this method

https://steamcommunity.com/sharedfiles/filedetails/?id=1086279994

it broke between 1.8.1 and 1.8.2 here is the manifest ids i used

17 sep 1.8.2 (broken)
250820 250821 3738881457872358261
250820 250827 7405209456960741105
250820 250824 6413409665598930737
250820 250830 8255214014436030582
250820 250833 7725354146120744174
250820 250831 8065879212735109331

16 sep 1.8.1 (working)
250820 250821 539049345897333342
250820 250827 7405209456960741105
250820 250824 4615817992650268032
250820 250830 8255214014436030582
250820 250833 7725354146120744174
250820 250831 8065879212735109331

**Comment by AndersMalmgren** [comment]
I just tried 1.9.7 and it fixes my issues with Vive. But now there are problems with Knuckles instead :/ If you try my repro from earlier with Knuckles then GetAxis will return what  ever values was present when you call Activate. So after Activate is called GetAxis will return same value no matter what actual value GetAxis should have

**Comment by AndersMalmgren** [comment]
This is interesting, in 1.8.2 (the version were problems frist happened) I get the same behaviour as with Vive Wands. in 1.9.7 or in 1.8.21 I get above described behavior.

**Comment by zite** [comment]
This has been fixed in SteamVR Beta 1.9.12
